test/benchmarks/Benchmarks.Trace/AgentWriterBenchmark.cs

```diff
@@ -104,11 +103,11 @@ public void AddHeader(string name, string value)
                 _realRequest.AddHeader(name, value);
             }
 
-            public async Task<IApiResponse> PostAsync(Span[][] traces, FormatterResolverWrapper formatterResolver)
+            public async Task<IApiResponse> PostAsync(ArraySegment<byte> traces)
             {
                 using (var requestStream = Stream.Null)
                 {
-                    await CachedSerializer.Instance.SerializeAsync(requestStream, traces, formatterResolver).ConfigureAwait(false);
+                    await requestStream.WriteAsync(traces.Array, traces.Offset, traces.Count).ConfigureAwait(false);
```

Comment:
Are there relevant benchmark numbers we can include in the PR description?

Reply:
Done